src/app/games/pentago/pentago.didacticial.ts

```diff
@@ -34,10 +34,11 @@ export const pentagoDidacticial: DidacticialStep[] = [
         ], 10),
     ),
```

Comment:
no longer up to date

Reply:
hmm quoi donc ? Le texte et les plateaux semblent matcher l'autre branche ? Et le `pentagoDidacticial` est bien changé en `pentagoTutorial`

Reply:
Y a withPreviousMove en plus !

Reply:
Ah oui dans le dernier commit, j'attends que tu aies pull `develop` dans la branche en question et régler les confilts avant de l'intégrer à celle-ci !

Reply:
nu ça devrait être bon